Bug Report: TestAccessLog failed in unittests workflow

**What happened**:

The `unittests` workflow failed in job `75527170628` for `juicedata/juicefs`.

The actual failing test in the log is `TestAccessLog` from `pkg/vfs`, with this error:

`accesslog_test.go:65: unexpected log: " [uid:1,gid:2,pid:10] method test - OK <0.000157>\n"`

The job then ended with:

- `FAIL github.com/juicedata/juicefs/pkg/vfs`
- `make: *** [Makefile:118: test.pkg] Error 1`
- `Process completed with exit code 2.`

Although the log contains multiple `--- ERROR ---` markers earlier, many of those sections are followed by passing tests. The definitive failure causing the workflow to fail is `TestAccessLog`.

**What you expected to happen**:

The `unittests` workflow should pass, and `TestAccessLog` should match the expected access log output.

**How to reproduce it (as minimally and precisely as possible)**:

1. Check out commit `d01b3829aa143b850a02c92357c7317a050e4d23`
2. Run the unit tests for the repository
3. Observe the failure in `pkg/vfs`, especially `TestAccessLog`

Relevant failing excerpt:

- `accesslog_test.go:65: unexpected log: " [uid:1,gid:2,pid:10] method test - OK <0.000157>\n"`
- `FAIL github.com/juicedata/juicefs/pkg/vfs`

Workflow file:
- `.github/workflows/unittests.yml`

**Anything else we need to know?**

- Failing job ID: `75527170628`
- Workflow commit: `d01b3829aa143b850a02c92357c7317a050e4d23`

This looks related to access log formatting or a mismatch between actual log output and test expectations in `pkg/vfs/accesslog_test.go`.

**Environment**:
- JuiceFS version (use `juicefs --version`) or Hadoop Java SDK version: Unknown from CI log
- Cloud provider or hardware configuration running JuiceFS: GitHub Actions runner
- OS (e.g `cat /etc/os-release`): Unknown from provided job log
- Kernel (e.g. `uname -a`): Unknown from provided job log
- Object storage (cloud provider and region, or self maintained): Not required to identify this failing test from the CI log
- Metadata engine info (version, cloud provider managed or self maintained): Unknown from provided job log
- Network connectivity (JuiceFS to metadata engine, JuiceFS to object storage): Unknown from provided job log
- Others:
  - Workflow: `unittests`
  - Repository: `juicedata/juicefs`

--- a/pkg/vfs/accesslog_test.go
+++ b/pkg/vfs/accesslog_test.go
@@ -17,7 +17,7 @@
 package vfs
 
 import (
-	"regexp"
+	"strings"
 	"testing"
 	"time"
 
@@ -62,8 +62,7 @@ func TestAccessLog(t *testing.T) {
 	if now.Sub(ts.Local()) > time.Millisecond*10 {
 		t.Fatalf("stale time: %s now: %s", ts, time.Now())
 	}
-	matched, _ := regexp.MatchString(` $$uid:1,gid:2,pid:10$$ method test - OK <0\.\d{4,6}>`, logs[26:])
-	if !matched {
+	if !strings.HasPrefix(logs[26:len(logs)-1], " [uid:1,gid:2,pid:10] method test - OK <0.") {
 		t.Fatalf("unexpected log: %q", logs[26:])
 	}
 

--- a/pkg/vfs/accesslog_test.go
+++ b/pkg/vfs/accesslog_test.go
@@ -17,6 +17,7 @@
 package vfs
 
 import (
+	"regexp"
 	"testing"
 	"time"
 
@@ -61,7 +62,8 @@ func TestAccessLog(t *testing.T) {
 	if now.Sub(ts.Local()) > time.Millisecond*10 {
 		t.Fatalf("stale time: %s now: %s", ts, time.Now())
 	}
-	if logs[26:len(logs)-4] != " [uid:1,gid:2,pid:10] method test - OK <0.0000" {
+	matched, _ := regexp.MatchString(` $$uid:1,gid:2,pid:10$$ method test - OK <0\.\d{4,6}>`, logs[26:])
+	if !matched {
 		t.Fatalf("unexpected log: %q", logs[26:])
 	}
 

